spring-boot3-dummy/src/main/java/org/springframework/boot/testcontainers/service/connection/ContainerConnectionSource.java

```diff
@@ -0,0 +1,7 @@
+package org.springframework.boot.testcontainers.service.connection;
```

Comment:
I'm missing the copyright notice, which I assume isn't even going to be our default copyright notice, right?